css/badge.css

```diff
@@ -0,0 +1,76 @@
+﻿.badge-element{
+    position: relative;
+    
+}
+
+.badge-icon {    
+    background-color: var(--info-text);
+    border-radius: 50rem;
+    color: var(--white-color);
+    display: inline-block;
+    font-size: 0.75rem;
+    font-weight: 600;
+    padding: 0.35em 0.5em;
+    text-align: center;
+    line-height: 1;
+    position: absolute;
+    left: 70%;
+    top: 0;
+    transform: translate(-50%, -50%);
+    white-space: nowrap;
+}
+
+.badge-red{
+    background-color: red;
```

Comment:
Please use hex codes for consistency and if the same color is repeated, declare a variable and use it.